build/BuildSteps.cs

```diff
@@ -425,15 +425,6 @@ public static void Zip()
                 ColoredConsole.WriteLine($"Creating {shaPath}");
                 File.WriteAllText(shaPath, ComputeSha256(zipPath));
 
-                try
-                {
-                    Directory.Delete(path, recursive: true);
```

Comment:
Why did we remove this?

Reply:
I imagine it's removed to be able to use the artifacts for MSI creating after. Maybe at the end of the powershell script, we should delete this directory?

Reply:
Leaving this as a future enhancement for now.

Reply:
This has been done.